benchmarking/app/src/main/AndroidManifest.xml

```diff
@@ -1,17 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
-  Copyright 2022 The Android Open Source Project
```

Comment:
Something's off in this formatting.

Reply:
Okay, I reverted and mimicked the copyrights we had in the project so there's not so many changes.